runtime/compiler/runtime/asmprotos.h

```diff
@@ -143,6 +143,9 @@ JIT_HELPER(jitThrowCurrentException);  // asm calling-convention helper
 JIT_HELPER(jitThrowException);  // asm calling-convention helper
 JIT_HELPER(jitThrowUnreportedException);  // asm calling-convention helper
 JIT_HELPER(jitThrowExceptionInInitializerError);  // asm calling-convention helper
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+JIT_HELPER(jitThrowIdentityException);  // asm calling-convention helper
+#endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
```

Comment:
Same comment as the above regarding to J9VM_OPT_VALHALLA_VALUE_TYPES.

Reply:
Same as the previous comment - I think I had left this `ifdef` in because of the way I defined `old_slow_jitThrowIdentityException` in an earlier version of the code.  I will remove it.

Reply:
Corrected in commit 18dd4ab4d51a30c7e44bac5052fa00b535cec694